<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DuckStation Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resources.qrc">
    <normaloff>:/icons/duck.png</normaloff>:/icons/duck.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="settingsCategory">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>230</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>General Settings</string>
      </property>
      <property name="icon">
       <iconset theme="GeneralSettings"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BIOS Settings</string>
      </property>
      <property name="icon">
       <iconset theme="BIOSSettings"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Console Settings</string>
      </property>
      <property name="icon">
       <iconset theme="ConsoleSettings"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Emulation Settings</string>
      </property>
      <property name="icon">
       <iconset theme="EmulationSettings"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Game List Settings</string>
      </property>
      <property name="icon">
       <iconset theme="GamelistSettings"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hotkey Settings</string>
      </property>
      <property name="icon">
       <iconset theme="HotkeySettings"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Controller Settings</string>
      </property>
      <property name="icon">
       <iconset theme="ControllerSettings"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Memory Card Settings</string>
      </property>
      <property name="icon">
       <iconset theme="MemorycardSettings"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Display Settings</string>
      </property>
      <property name="icon">
       <iconset theme="DisplaySettings"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Enhancement Settings</string>
      </property>
      <property name="icon">
       <iconset theme="EnhancementSettings"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Post-Processing Settings</string>
      </property>
      <property name="icon">
       <iconset theme="PostprocessingSettings"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Audio Settings</string>
      </property>
      <property name="icon">
       <iconset theme="AudioSettings"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Achievement Settings</string>
      </property>
      <property name="icon">
       <iconset theme="AchievementsSettings"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Advanced Settings</string>
      </property>
      <property name="icon">
       <iconset theme="AdvancedSettings"/>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="settingsContainer">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTextEdit" name="helpText">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="restoreDefaultsButton">
       <property name="text">
        <string>Restore Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
